lab_1_classify_by_unigrams/main.py

```diff
@@ -105,6 +189,29 @@ def detect_language(
 
     In case of corrupt input arguments, None is returned
     """
+    if unknown_profile is None or profile_1 is None or profile_2 is None:
+        return None
+    if (not isinstance(unknown_profile, dict) or not isinstance(profile_1, dict)
+            or not isinstance(profile_2, dict)):
+        return None
+
+    name1 = profile_1.get('name')
+    name2 = profile_2.get('name')
+    mse_1 = compare_profiles(unknown_profile, profile_1)
+    mse_2 = compare_profiles(unknown_profile, profile_2)
+    result1 = str(name1)
+    result2 = str(name2)
+
+    if mse_1 is None or mse_2 is None:
+        return None
+
+    if mse_1 < mse_2:
+        return result1
+    if mse_1 == mse_2:
+        profile_sorted_list = sorted([result1, result2])
+        return str(profile_sorted_list[0])
```

Comment:
the same

Reply:
I don't really get how I can remove str here if the variable profile_sorted_list is a list, and I need to get str as the result of the function.